data/src/main/java/org/oppia/android/data/backends/gae/model/GaeAnswerGroup.kt

```diff
@@ -5,7 +5,7 @@ import com.squareup.moshi.JsonClass
 
 /**
  * Data class for AnswerGroup model
- * https://github.com/oppia/oppia/blob/15516a/core/domain/state_domain.py#L43
+ * @link https://github.com/oppia/oppia/blob/develop/core/domain/state_domain.py#L67
```

Comment:
I think per https://stackoverflow.com/a/45195371 KDoc doesn't use ``@link`` since Markdown automatically supports linkification of URLs.

Reply:
Yes, I saw this earlier actually and didn't find any option where we can get right-click on the link to open it, so I simply put a link annotate

Reply:
Doesn't right clicking just work for links in KDocs? Alternatively, you could use bracket notation & name the links. Either way, ``@link`` is actually invalid for KDoc, so we should remove it.

Reply:
this is something different I found. 
If we keep links in `// ...link...` then it is clickable, but we have link inside the block `/* ...link... */` then it is not.

Example
![Screenshot 2021-02-02 at 09 03 33](https://user-images.githubusercontent.com/25057618/106548743-90af1f00-6535-11eb-8d64-df8cf0d2e0e9.png)